src/helm/benchmark/scenarios/cleva_scenario.py

```diff
@@ -519,6 +519,12 @@ def process_dialogue_instance(self, row: Dict[str, Any], split: str) -> List[Ins
 class CLEVASubjectKnowledgeScenario(CLEVAScenario):
     """
     The subject knowledge task of CLEVA benchmark.
+    We follow HELM (https://github.com/stanford-crfm/helm/tree/main/scripts/fact_completion) to construct the dataset.
+    Considering the Chinese characteristics, we rewrite and extend the relations.
+    There are 91 relations, which are grouped into 14 subjects:
```

Comment:
I think we do not need to explicitly inform others that there are 91 relations, as we might include more in the future and the current released version does not have 91 relations.